### PR TITLE
Windows Compile fails due to missing Display#isXMouseActive

### PR DESCRIPTION
**Pull request by niraj-modi**
eclipse-platform#98

Change-Id: Id36bd8491165e645334cddba3fb6b30fc745ea8e
Signed-off-by: Niraj Modi <niraj.modi@in.ibm.com>

### CODE REVIEW

File: tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/tests/win32/AllWin32Tests.java [hunk 1]
```diff
@@ -23,7 +22,7 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
 	Test_org_eclipse_swt_dnd_DND.class,
-	Test_org_eclipse_swt_widgets_Display.class,
+//	Test_org_eclipse_swt_widgets_Display.class,
```
Comment (laeubi) [inline tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/tests/win32/AllWin32Tests.java:25]: Just wondering as it is named `AllWin32Tests` should it not actually *compile* when compiling for/with windows?

**Comment by laeubi** [comment]
Just to not confuse here I just want to compile all sources/test not running them (yet), so it seems the compiler here gets a "wrong" reference to compile the test. As also the running later fails, I assume there is something missing in the maven workflow currently so this seems a good first start to verify windows-compile will use the windows compiled sources.

**Comment by laeubi** [comment]
@niraj-modi currently you disable the **execution** of the test, but here the **compilation** fails! @akurtakov do you have any idea what could be wrong here? I have now removed all but the compilation phase but it seems the test compile is pulling in the linux part instead of the windows part ...

**Comment by niraj-modi** [comment]
> @niraj-modi currently you disable the **execution** of the test, but here the **compilation** fails! @akurtakov do you have any idea what could be wrong here? I have now removed all but the compilation phase but it seems the test compile is pulling in the linux part instead of the windows part ...

For now I could work-around this compilation failure using reflection.

**Pull request merged**